lottie/src/main/java/com/airbnb/lottie/CompositionLayer.java

```diff
@@ -106,4 +108,16 @@ boolean hasMatte() {
     }
     return false;
   }
+
+  @Override
+  public void addColorFilter(@Nullable String layerName, @Nullable String contentName,
```

Comment:
@seanfchan You can nest compositions (called precomps). That will create a CompositionLayer inside of the original CompositionLayer. You should be able to specify the precomp layer as the layer to get tinted and then it should tint all layers within the precomp. Does that make sense?